source/draw/Color.ooc

```diff
@@ -137,6 +142,31 @@ ColorYuv: cover {
 	operator != (other: ColorBgra) -> Bool { !this equals(other) }
 }
 
+ColorYuva: cover {
+	yuv: ColorYuv
+	alpha: UInt8
+	init: func@ (=yuv, =alpha)
+	init: func@ ~uint8 (y, u, v, a: UInt8) {
+		this yuv = ColorYuv new(y, u, v)
+		this alpha = a
+	}
+	init: func@ ~default { this init(0, 128, 128, 255) }
```

Comment:
@simonmika you talked about this recently, and I seem to remember you saying that a default constructor shouldn't initialize to anything but 0. What should we do in a case like this?